File: openapi/code_samples/javascript/api_company_filter.js
const express = require('express');
const app = express();
const port = 3000;

app.use(express.json());

app.get('/api/company/list-indicators', (req, res) => {
    const companyFilterCriteria = req.query.companyFilterCriteria;
    // Your implementation here
    res.status(200).json({});
});

app.listen(port, () => {
    console.log(`Server is listening at http://localhost:${port}`);
});

File: openapi/code_samples/java/api_company_filter.java
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestParam;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class CompanyController {

    @GetMapping("/api/company/list-indicators")
    public ResponseEntity<?> listIndicators(@RequestParam CompanyFilterCriteria companyFilterCriteria) {
        // Your implementation here
        return ResponseEntity.ok().build();
    }
}
